extensions/scarb-doc/src/types.rs

```diff
@@ -23,12 +23,14 @@ use crate::db::ScarbDocDatabase;
 #[derive(Serialize, Clone)]
 pub struct Crate {
     pub root_module: Module,
+    pub name: String,
```

Comment:
```suggestion
    pub name: String,
    pub root_module: Module,
```

wouldn't this be easier to process by humans in generated JSON?

Reply:
Agree